### DISCUSSION TITLE
中英双语互译

### DISCUSSION THREAD

**Discussion by dongliang** [immersive-translate/immersive-translate#422] [Ideas]
### 新功能 | New Feature

1.开启这个功能后，网页中的源语言会翻译为目标语言。
2.同时，目标语言会翻译为源语言。
3.中英双语的网页，可以将全部文字翻译为双语，而不需要频繁切换语言。

### 为什么您想要添加这些功能 | Why you want to add these features

1.不想每次都切换目标语言。中英互译
2.
3.

### 提议 | Proposal

应该在调用接口之前，判断如果为目标语言则翻译为源语言。我有兴趣开发，但是不确能自己是否有这个能力。

**Comment by theowenyoung** [comment]
沉浸式翻译目前的主要目的是加速信息的获取，这个功能会有点 Confuse, 所以暂时不会考虑增加这个功能。
